### ISSUE TITLE
ModelEMA update

### ISSUE THREAD

**Issue by zwhua006** [ultralytics/yolov5#5240]
## Additional context
 def update(self, model):
        # Update EMA parameters
        with torch.no_grad():
            self.updates += 1
            d = self.decay(self.updates)
            msd = model.module.state_dict() if is_parallel(model) else model.state_dict()  # model state_dict
            for k, v in self.ema.state_dict().items():
                if v.dtype.is_floating_point:
                    v *= d
                    v += (1. - d) * msd[k].detach()

In this function,items() means the copy of self.ema.state_dict().If you update the parameter v directly instead of using self.ema.state_dict()[k] = v,the result will be computed from pretrained weights's ema. It's not from last ema. I want to know how it works.
